code/modules/organs/internal/internal_organ_processes.dm

```diff
@@ -0,0 +1,212 @@
+#define BRUISED_2_EFFICIENCY 80
+#define BROKEN_2_EFFICIENCY 50
+#define DEAD_2_EFFICIENCY 0
+
+//Has processes for all internal organs, called from /mob/living/carbon/human/Life()
+
+/mob/living/carbon/human/proc/porcess_internal_ograns() //Calls all of the internal organ processes
+	if(should_have_process(OP_EYES))	//Bad way to do this, should be reworked to call procs from a list that's generated by the species.has_process list + extra processes gotten from organs (carrion)
+		eye_process()
+	if(should_have_process(OP_KIDNEYS))
+		kidney_process()
+	if(should_have_process(OP_LIVER))
+		liver_process()
+	if(should_have_process(OP_HEART))
+		heart_process()
+	if(should_have_process(OP_LUNGS))
+		lung_process()
+	if(is_carrion(src))
+		carrion_process()
+
+/mob/living/carbon/human/proc/get_organ_efficiency(process_define)
+	var/list/process_list = internal_organs_by_efficiency[process_define]
+	var/effective_efficiency = 0
+	if(process_list.len)
+		for(var/organ in process_list)
+			var/obj/item/organ/internal/I = organ
+			effective_efficiency += I.organ_efficiency[process_define] - (I.organ_efficiency[process_define] * (I.damage / I.max_damage))
```

Comment:
I would like this line of code to be a proc of the organs themselves, in case someone wants to make different efficiencies for different organs and it is also easier to add new code.

Reply:
Fixed.